index.html

```diff
@@ -0,0 +1,170 @@
+            <h3 class="programheadtext">OPENING</h3>
+            <p class="programtext">
+              During th opening we have a moment where veryone can quetly get to know each other, just talk to people that see interesting, you might make new friends.
+            </p>
+          </li>
+          <li>
+            <img class="progicon" src="assets/images/programicon2.png" alt="icon" />
+            <h3 class="programheadtext">Lectures</h3>
+            <p class="programtext">
+              At 7 P.M. we start with lectures by our top speakers, they will discuss new developments in the world of chess, we made sure playes from all skill lvels will be able to understand
+            </p>
+          </li>
+          <li>
+            <img class="progicon" src="assets/images/programicon3.png" alt="icon" />
+            <h3 class="programheadtext">Dialoque</h3>
+            <p class="programtext">
+              Have that game that is still stuck in your mind from months ago? This is the time to talk about with playes who can provide you with all sorts of perspective
+            </p>
+          </li>
+          <li>
+            <img class="progicon" src="assets/images/programicon4.png" alt="icon" />
+            <h3 class="programheadtext">PLANS</h3>
+            <p class="programtext">
+              The next next agenda point is the discussing of strategies, are there any new opening moves or strategies, or is it still the old timers who have the advantage.
+            </p>
+          </li>
+          <li>
+            <img class="progicon" src="assets/images/programicon5.png" alt="icon" />
+            <h3 class="programheadtext">MINGLE</h3>
+            <p class="programtext">
+              Personally, this is always the main advantage of these kind of meetups for me, i learn so much from all the different eople i meet during theese events.
+            </p>
+          </li>
+        </ul>
+      </div>
+      <div class="programlinkwrapper">
+        <a class="programlink" href="#">SEE THE WHOLE PROGRAM</a>
+      </div>
+    </section>
+    <section class="speakerssect">
+      <div class="announce">
+        <div class="announcetext">Featured speakers</div>
+        <hr class="announcehr" />
+      </div>
+      <div class="speakers"></div>
+    </section>
+    <section class="partnerssec">
+        <div class="announce">
+            <div class="announcetext">Partner</div>
+            <hr class="announcehr" />
+          </div>
+     <div class="partnerwrapper">
+         <ul class="partners">
+             <li><img src="assets/images/navar.png" alt=""></li>
+             <li><img src="assets/images/duamkako.png" alt=""></li>
+             <li><img src="assets/images/Google.png" alt=""></li>
+             <li><img src="assets/images/airbnb.png" alt=""></li>
+             <li><img src="assets/images/mozilla.png" alt=""></li>
+         </ul>
+     </div>
```

Comment:
- [ ] The view for the partner is too big in the mobile view, please correct it.

![mob-foot](https://user-images.githubusercontent.com/61562626/137174301-449bd448-14b9-48aa-846d-f0e3a4064f0c.jpeg)